src/main/resources/application.yaml

```diff
@@ -17,7 +17,7 @@ spring:
   application:
     name: Send Letter Provider
   datasource:
-      url: jdbc:postgresql://${LETTER_TRACKING_DB_HOST:send-letter-producer-database}:${LETTER_TRACKING_DB_PORT:5432}/${LETTER_TRACKING_DB_NAME:letter_tracking}
+      url: jdbc:postgresql://${LETTER_TRACKING_DB_HOST:send-letter-producer-database}:${LETTER_TRACKING_DB_PORT:5432}/${LETTER_TRACKING_DB_NAME:letter_tracking}${LETTER_TRACKING_DB_CONN_OPTIONS}
```

Comment:
I think you will also need to update Docker and related shell script files

Reply:
done - updated run-flyway.sh